app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,39 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.boxed {
+  border: 3px solid green ;
+  width: 60%;
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+section {
+  border: 3px solid green ;
+  margin-left: 250px;
+  margin-right: auto;
+  padding: 5px 10px 10px 5px;
+  width: 20%;
+}
+
+.posts {
+  border: 3px solid green ;
+  margin-right: 250px;
+  margin-left: auto;
+  padding: 5px 10px 10px 5px;
+  width: 20%;
+}
```

Comment:
@sharadalt PS. I recommend viewing **all** feedback on GitHub directly - the emails aren't very good and don't show the lines I'm referring to. https://github.com/sharadalt/blocipedea/pull/1/files